customizations/autoscaling/config.yaml

```diff
@@ -0,0 +1,112 @@
+accountsWww:
+  ## accounts-www autoscaling configuration
+  ## ref: https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/
+  ## @param accountsWww.autoscaling.enabled Enable autoscaling for the accounts-www containers
+  ## @param accountsWww.autoscaling.minReplicas The minimal number of containters for the accounts-www deployment
+  ## @param accountsWww.autoscaling.maxReplicas The maximum number of containers for the accounts-www deployment
+  ## @param accountsWww.autoscaling.targetCPUUtilizationPercentage The CPU utilization percentage used for scale up containers in accounts-www deployment
+  ##
+  autoscaling:
+    enabled: true
+    minReplicas: 1
```

Comment:
I would put this value and the following ones (`minReplicas`, `maxReplicas` & `targetCPUUtilizationPercentage`) as commented so that the customer who wants to change them but if not continue using our default ones.

That apply to all the file.